Create reproduction documentation + flakes

rbozan opened the pull request.

comzeradd commented:
It looks like it fails when it can’t find the Elasticpress plugin in order to deactivate it (we deactivate it on local developerment. The plugin should live inside `planet4/plugins`.

As a workaround you could try commenting the [respective line](https://github.com/rbozan/planet4-develop/blob/rbozan-patch-1/scripts/install.js#L90).

rbozan commented:
The installation process finishes but going to http://www.planet4.test/ just shows a 404 page. Logging in to WP Admin http://www.planet4.test:8889/wp-admin with `admin` / `admin` or `admin@planet4.test` / `admin` neither works, but I did see this in the logs:

```
ℹ Starting 'wp config set DB_NAME planet4_dev' on the cli container. 

Success: Updated the constant 'DB_NAME' in the 'wp-config.php' file with the value 'planet4_dev'.
✔ Ran `wp config set DB_NAME planet4_dev` in 'cli'. (in 2s 356ms)
ℹ Starting 'wp user create admin admin@planet4.test --user_pass=admin --role=administrator' on the cli container. 

Error: The 'admin' username is already registered.
✖ Command failed with exit code 1
Command failed with exit code 1
ℹ Starting 'wp user update admin --user_pass=admin --user_email=admin@planet4.test --role=administrator' on the cli container. 

sendmail: can't connect to remote host (127.0.0.1): Connection refused
sendmail: can't connect to remote host (127.0.0.1): Connection refused
Success: Updated user 1.
✔ Ran `wp user update admin --user_pass=admin --user_email=admin@planet4.test --role=administrator` in 'cli'. (in 2s 525ms)
ℹ Starting 'wp plugin activate --all' on the cli container. 

Plugin '{nix-shell-env}' activated.
Success: Activated 1 of 18 plugins.
✔ Ran `wp plugin activate --all` in 'cli'. (in 1s 65ms)
ℹ Starting 'wp plugin install query-monitor' on the cli container. 

Installing Query Monitor – The developer tools panel for WordPress (3.16.2)
Downloading installation package from https://downloads.wordpress.org/plugin/query-monitor.3.16.2.zip...
The authenticity of query-monitor.3.16.2.zip could not be verified as no signature was found.
Unpacking the package...
Installing the plugin...
Plugin installed successfully.
Success: Installed 1 of 1 plugins.
✔ Ran `wp plugin install query-monitor` in 'cli'. (in 5s 494ms)
The local instance is now available at http://www.planet4.test/
```

Also did not intend to open a PR so sorry about that.
